zingolib/src/lightclient/send.rs

```diff
@@ -137,25 +137,49 @@ impl LightClient {
 #[cfg(feature = "zip317")]
 /// patterns for newfangled propose flow
 pub mod send_with_proposal {
```

Comment:
i cant see this mod in the changelog, if its added all other pub in this mod will be covered implicitly

Reply:
Perhaps....but this PR adds no public methods to the interface.

Reply:
well my point is it changes `pub enum CompleteAndBroadcastError` and therefore the quest of how do i update the changelog should resolve at "oops this whole mod was not added in the first place"

Reply:
Oh right, the error is public. D'oh.